recipes/gemserver.rb

```diff
@@ -0,0 +1,39 @@
+
+case node['platform_family']
+when "debian"
+  bash "chefdk-install-debian" do
+    user "root"
+    code <<-EOH
+        cd /tmp
+        yum install wget -y
+        wget -nc #{node['download_url']}/#{node['setup']['chefdk']['debian']}
+        apt install #{node['setup']['chefdk']['debian']} -y
+      EOH
+  end
+
+when "rhel"
+  bash "chefdk-install-rhel" do
+    user "root"
+    code <<-EOH
+        cd /tmp
+        yum install wget -y
+        wget -nc #{node['download_url']}/#{node['setup']['chefdk']['rhel']}
+        yum install -y #{node['setup']['chefdk']['rhel']} 
+      EOH
+  end
+
+end
```

Comment:
This part doesn't make much sense. We run setup-chef with Karamel, which as first step already downloads and install chefdk. So by the time we get here, chefdk is already installed. 

Additionally yum will fail if the package you are trying to install already exists. So this whole recipe is going to fail

Reply:
This didn't work. It's been done in karamel. Closing.